modules/candidate_list/jsx/candidateListIndex.js

```diff
@@ -155,7 +171,8 @@ class CandidateListIndex extends Component {
         show: false,
```

Comment:
visit label is currently set to not 'show' in the data table so what you added in the format column for column === 'Visit Label' won't get implement

Reply:
oh true! good point